ydb/library/yql/tests/sql/suites/blocks/member.sql

```diff
@@ -0,0 +1,8 @@
+USE plato;
+/* XXX: Enable UseBlocks pragma and provide input to trigger block execution. */
+pragma UseBlocks;
+
+SELECT
+    TryMember(val, "a", 999) as a,
+    TryMember(val, "b", 999) as b,
```

Comment:
we don't need to test block version of TryMember at all

Reply:
Agree, removed.